validphys2/src/validphys/closuretest/multiclosure_nsigma_helpers.py

```diff
@@ -0,0 +1,170 @@
+"""
+This module contains some helper functions that are used for the computation of nsigma in the
+context of a multi-closure test.
+"""
+
+import dataclasses
+import pandas as pd
+import numpy as np
+import logging
+
+from reportengine import collect
+
+from validphys.core import DataSetSpec, PDF
+from nnpdf_data.coredata import CommonData
+from validphys.calcutils import calc_chi2
+from validphys import convolution
+
+log = logging.getLogger(__name__)
+
+
+@dataclasses.dataclass
+class CentralChi2Data:
+    value: float
+    ndata: int
+    dataset: DataSetSpec
+
+    @property
+    def reduced(self):
+        return self.value / self.ndata
+
+
+def central_predictions(dataset: DataSetSpec, pdf: PDF) -> pd.DataFrame:
+    """
+    Computes the central prediction (central PDF member) for a dataset.
+
+    Parameters
+    ----------
+    dataset: validphys.core.DataSetSpec
+    pdf: validphys.core.PDF
+
+    Returns
+    -------
+    pd.DataFrame
+        index is datapoints, column is the central prediction.
+    """
+    return convolution.central_predictions(dataset, pdf)
```

Comment:
```suggestion
from convolution import central_predictions
```

would be the same. You can add the type hints there if you need them.